### PR TITLE
Redirect links to new article location

### PR DESCRIPTION
**Pull request by estelle**
Article has been moved; redirect links to the new location and update content to better reflect where the link goes.

### CODE REVIEW

File: files/en-us/web/media/images/index.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ These articles provide guidance on selecting and configuring image types.
 
 - [Image file type and format guide](/en-US/docs/Web/Media/Formats/Image_types)
   - : A guide to the various image file types commonly supported by web browsers including details about their individual use cases, capabilities, and compatibility factors. Additionally, this article provides guidance for selecting the best image file type for a given situation.
-- [Mapping the width and height attributes of media container elements to their aspect-ratio](/en-US/docs/Web/Media/images/aspect_ratio_mapping)
-  - : This changes how browsers load images so their aspect ratios are calculated by the browser early on and can later be used to correct the display size of an image before it's loaded.
+- [Including `width` and `height` attributes helps avoid jank](/en-US/docs/Learn/Performance/Multimedia#rendering_strategy_preventing_jank_when_loading_images)
+  - : This changes how browsers load images so their aspect ratios are calculated by the browser early on and can later be used to reserves the display size of an image before it's loaded.
```
Comment (dipikabh) [inline files/en-us/web/media/images/index.md:42]:
```suggestion
  - : This changes how browsers load images so their aspect ratios are calculated by the browser early on and can later be used to reserve the display size of an image before it's loaded.
```

Comment (dipikabh) [inline files/en-us/web/media/images/index.md:42]:
```suggestion
  - : This changes how browsers load images so their aspect ratios are calculated by the browser early on and can later be used to reserve the display size of an image before it's loaded.
```

**Review by dipikabh — CHANGES_REQUESTED** [review]

**Comment by estelle** [comment]
That can be done in a different PR. This PR was just to get rid of links to a page that no longer exists.

**Review by dipikabh — APPROVED** [review]
Thanks!

**Pull request merged**